lib/src/repo.rs

```diff
@@ -1169,7 +1169,8 @@ impl MutableRepo {
         let heads_to_add = heads_to_add_expression
             .evaluate_programmatic(self)
             .unwrap()
-            .iter();
+            .iter()
+            .map(Result::unwrap);
```

Comment:
nit: same here (and two more changes in repo.rs)
The error should have to be returned to caller. TODO comment?